Installers/Mac/Setup/Contents/MacOS/minima

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# Constants
+JAVA_MINIMUM=11
+APP_JAR="minima.jar"
+APP_NAME="Minima"
+VM_ARGS="-rpcenable true"
+MINIMA_FOLDER=$HOME/.minima
+
+# Set the working directory
+DIR=$(cd "$(dirname "$0")"; pwd)
+
+# Error message for NO JAVA dialog
+ERROR_TITLE="Cannot launch $APP_NAME"
+ERROR_MSG="$APP_NAME requires Java version $JAVA_MINIMUM or later to run."
+DOWNLOAD_URL="https://www.oracle.com/uk/java/technologies/javase/jdk11-archive-downloads.html"
+
+# Is Java installed?
+if type -p java; then
+    _java="java"
+elif [[ -n "$JAVA_HOME" ]] && [[ -x "$JAVA_HOME/bin/java" ]]; then
+    _java="$JAVA_HOME/bin/java"
+else
+    osascript \
+	-e "set question to display dialog \"$ERROR_MSG\" with title \"$ERROR_TITLE\" buttons {\"Cancel\", \"Download\"} default button 2" \
```

Comment:
is this a dialog box that shows up?

Reply:
That's correct, if JAVA_HOME couldn't find a popup would have been displayed. Removed in: https://github.com/minima-global/Minima/pull/570/